features/dom.yml

```diff
@@ -148,6 +150,7 @@ compat_features:
   - api.Element.append
   - api.Element.attributes
   - api.Element.before
+  - api.Element.beforematch_event
```

Comment:
Reading the MDN docs for this event, it seems like it belongs to hidden-until-found instead.